apps/calculation/additional_outputs/integer_list_controller.cpp

```diff
@@ -47,9 +69,44 @@ I18n::Message IntegerListController::messageAtIndex(int index) {
       return I18n::Message::HexadecimalBase;
     case 2:
       return I18n::Message::BinaryBase;
+    case 3:
+        return I18n::Message::Scientific;
```

Comment:
Can you fix indentation ?
```suggestion
      return I18n::Message::Scientific;
```